en/docs/apis/api-concepts.md

```diff
@@ -1,23 +1,70 @@
 # API Concepts
-Before you start managing APIs via Choreo, there are basic concepts that you need to understand. This section walks you through all the concepts you need to know. 
+Choreo's API management capabilities allow you to create APIs, manage APIs, and expose services and external APIs as managed APIs. Here are some basic concepts which will help you understand API management in Choreo as you create and manage your APIs in the [Choreo Console]({{choreo_console}}).
  
 ## API
-An API in Choreo allows you to expose business functionality as a service to customers, partners, suppliers, and other stakeholders.
+An API is an intermediate layer that acts as a communication protocol between a consumer and a service, simplifying the consumption of the service. In addition to hiding the underlying implementation details of a service, an API provides a secure, controlled, and well-documented approach to access an exposed service.
+
+## API Format
+Open API Specification (OAS, a.k.a Swagger) format is the underlying representation of an API in Choreo. You can import an API using a Swagger definition in the Publisher Portal. Additionally, a user can view, edit, import, or download an API definition in OAS format via the API Publisher Portal.
+
+## API Resource path and HTTP Methods
+An API comprises one or more resources, each of which has a unique resource path (URI). An API Resource has a set of HTTP methods that operates on it. The supported HTTP methods are GET, POST, PUT, DELETE, PATCH, HEAD, and OPTIONS.
+
+## API Lifecycle
+API lifecycle comprises of the stages an API goes through from creation to retirement. API lifecycle is independent of its backend service. The lifecycle states are `Created`, `Prototyped`, `Published`, `Deprecated`, `Blocked`, `Deployed`, and `Retired`.
+
+## Application
+An Application in Choreo is the logical representation of a physical application such as a mobile app, web app, device, etc. For an application to use an API, the application should subscribe to the relevant APIs it intends to use. A subscription to an API happens over a selected business plan.  The business plan determines the usage quota the application gets. An application has a consumer key and a consumer secret which acts as the credentials of the application.
+
+## Access Token
+Choreo uses the OAuth2.0 bearer token-based authentication mechanism to allow a consumer to access an API.  
+The access token is a string that the consumer passes in an HTTP header of the API request. Choreo supports JWT formatted self-contained access tokens.
+
+## Rate Limiting
+Rate limiting allows you to limit the number of permitted requests to an API for a given time window. Rate limiting provides you to:
```

Comment:
Doesn't seem correct to say "Rate limiting provides you to:"
We might have to go with "Rate limiting allows you to:" here and perhaps consider changing the previous sentence as "Rate limiting makes it possible to limit the number of . . . ."

Reply:
done